applications/openshift/kubelet/kubelet_configure_event_creation/tests/ocp4/e2e.yml

```diff
@@ -1,4 +1,3 @@
 ---
 # This rule fails by default on 4.13, and passes by default on 4.14
-default_result: FAIL
-result_after_remediation: PASS
+default_result: PASS
```

Comment:
Remove the comment above too, which becomes irrelevant once the rule is passing on supported OCPs